ao-contracts/curve-bonded-token/src/utils/mod.lua

```diff
@@ -0,0 +1,30 @@
+local bint = require('.bint')(256)
+
+local utils = {
+    add = function(a, b)
+        return tostring(bint(a) + bint(b))
+    end,
+    subtract = function(a, b)
+        return tostring(bint(a) - bint(b))
+    end,
+    multiply = function(a, b)
+        return tostring(bint(a) * bint(b))
+    end,
+    divide = function(a, b)
+        return tostring(bint(a) / bint(b))
```

Comment:
Pay close attention to this if you need integer division with `bint`s
This function might return a number that is non-parseable as `bint`